docs/examples/impress.values.yaml

```diff
@@ -33,8 +33,8 @@ backend:
     OIDC_RP_SIGN_ALGO: RS256
     OIDC_RP_SCOPES: "openid email"
     OIDC_VERIFY_SSL: False
-    USER_OIDC_FIELD_TO_SHORTNAME: "given_name"
-    USER_OIDC_FIELDS_TO_FULLNAME: "given_name,usual_name"
+    OIDC_USERINFO_SHORTNAME_FIELD: "given_name"
+    OIDC_USERINFO_FULLNAME_FIELDS: "given_name,usual_name"
```

Comment:
I don't know how to deal with it. Renaming it directly is a breaking change. Having both makes non sense has the new settings will overwrite the legacy one.